spec/rack/test/cookie_jar_spec.rb

```diff
@@ -39,6 +39,14 @@
     jar.get_cookie(cookie_name+'a').must_be_nil
   end
 
+  it '#get_cookie with a populated jar returns full cookie objects when passed a symbol' do
```

Comment:
I don't see any test for `#[]` with a symbol. We should add one.

Reply:
I can add one.